styles/components/_listingCard.scss

```diff
@@ -31,22 +36,10 @@
     align-items: center;
     justify-content: center;
     z-index: 1; // so it's above the overlay icons
-
-    .overlayPanelInner {
```

Comment:
I will bring this up with wolf because his removal of it from the html has caused some issue.